dashboard/modules/snapshot/snapshot_schema.json

```diff
@@ -181,7 +181,7 @@
                       "type": "integer"
                     },
                     "httpRoute": {
-                      "type": "string"
+                      "type": ["string", "null"]
```

Comment:
Sidenote (not relevant to this PR): there are a few other values listed as `string` in the snapshot value, which can actually be None. An example of this would be the deployment version. Currently these are serialized as `version or "None"`. Would it make sense to make these fields actually nullable?

Reply:
I think that would make a lot of sense -- do you want to file a follow-up issue/PR to clean this up?

Reply:
Most certainly!